product-mini/platforms/windows/CMakeLists.txt

```diff
@@ -151,6 +152,15 @@ set_target_properties (libiwasm PROPERTIES OUTPUT_NAME libiwasm)
 
 target_link_libraries (libiwasm ${LLVM_AVAILABLE_LIBS} ${UV_A_LIBS})
 
+
 if (MINGW)
   target_link_libraries (libiwasm ws2_32)
 endif ()
+
+if (WIN32)
+  target_link_libraries(libiwasm ntdll)
```

Comment:
I think ntdll should only be linked if WASI libc is enabled, otherwise there's no need for that.

Reply:
Should a check be added if WASI libc is enables

Reply:
I think the check should be (pseudocode):
if (WASI_LIBC is enabled AND MSVC is enabled)